packages/datatrak-web/src/api/mutations/useEditUser.ts

```diff
@@ -6,22 +6,34 @@
 import { useMutation, useQueryClient } from 'react-query';
 import { Project } from '@tupaia/types';
 import { put } from '../api';
+import { Entity } from '../../types';
 
-type ProjectId = { projectId: Project['id'] };
+type UserPreferences = { projectId?: Project['id']; countryId?: Entity['id'] };
 
 export const useEditUser = onSuccess => {
   const queryClient = useQueryClient();
 
-  return useMutation<any, Error, ProjectId, unknown>(
-    async ({ projectId }: ProjectId) => {
-      if (!projectId) {
+  return useMutation<any, Error, UserPreferences, unknown>(
+    async ({ projectId, countryId }: UserPreferences) => {
+      if (!projectId && !countryId) {
         return;
       }
 
+      const updates = {} as {
+        project_id?: Project['id'];
+        country_id?: Entity['id'];
+      };
+
+      if (projectId) {
+        updates.project_id = projectId;
```

Comment:
It's a shame that we have to convert these to snake case on the front end but there's nothing we can really do about it since we are not going to the orchestration server for this one